### ISSUE TITLE
Don't warn of unused variable which are made of underscores

### ISSUE THREAD

**Issue by Rizhiy** [microsoft/pyright#1836]
**Is your feature request related to a problem? Please describe.**
Pyright hints of unused variable for double underscore, e.g.:
```python
def func(*_, **__):
	pass
```
will hint that `__` is unused, which is false hint.

**Describe the solution you'd like**
If variable/argument name contains only underscores, unused hint should not be given.

**Additional context**
N/A

**Comment by Rizhiy** [comment]
Ok, information hint should not be given for variables which contain only underscores.
